style2.css

```diff
@@ -0,0 +1,404 @@
+ @media screen and (max-width: 1680px) {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)